frontend/src/components/v1/WalletStatusCard.tsx

```
@@ -0,0 +1,384 @@
+  const canConnect =
+    capabilities.canConnect && typeof onConnect === 'function';
+  const canDisconnect =
+    capabilities.isConnected && typeof onDisconnect === 'function';
+  const canRetry =
+    error !== null &&
+    error.recoverable &&
+    typeof onRetry === 'function';
+
+  // ── Handlers ─────────────────────────────────────────────────────────────────
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const handleConnect = useCallback(() => {
```

Biome (2.4.4)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)